app/views/csvs/new.html.erb

```diff
@@ -0,0 +1,42 @@
+<h1>Request a CSV Report</h1>
+
+<section class="clearfix">
+  <div class="csv-deploy-filters clearfix">
+    <%= form_tag csvs_path do %>
+      <div><strong>Deploy reports</strong></div>
+      <div>Configure your report with these parameters</div>
+      <div>
+        <%= label_tag :start_date, "Start Date:" %>
+        <%= select_date Date.today.prev_year, prefix: :start_date %>
```

Comment:
all these could be `filters[start_date]` etc and then we would send less 1-off things into the csv export columns